src/VirtualClient/VirtualClient.Contracts/VirtualClientIntervalBasedMonitor.cs

```diff
@@ -68,5 +69,17 @@ public TimeSpan MonitorWarmupPeriod
                     nameof(VirtualClientIntervalBasedMonitor.MonitorWarmupPeriod), VirtualClientIntervalBasedMonitor.defaultWarmupPeriod);
             }
         }
+
+        /// <summary>
+        /// The number of iterations with which this monitor runs
+        /// </summary>
+        public long MonitorIterations
```

Comment:
What's the reason to put a maximum when user doesn't define it? User might want infinite monitor iterations, like SAP

Reply:
I have set default value, reason behind it is if let say a monitor is using this to determine number of iterations and user removed the line which is providing this value in profile file then to protect the code from crashing, i gave a very high value which is like infinite only, do you see any concern?